Dockerfile

```diff
@@ -1,26 +1,50 @@
-FROM python:3.7.9-stretch
+ARG PY_IMG_TAG=3.7.9-stretch
+FROM python:${PY_IMG_TAG}
 
-RUN apt-get update \
-  && curl -sL https://deb.nodesource.com/setup_6.x | bash \
-  && apt-get install -y \
-    gettext curl build-essential nodejs ruby-sass ruby-compass \
-  && rm -rf /var/lib/apt/lists/* \
-  && pip install uwsgi \
-  && npm install -g yuglify
+# Set shell to use for run commands
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# Install repos for node6.x. 
+# WARNING: EOL on 2019-04-30
+# https://github.com/nodejs/Release#end-of-life-releases
+ARG NODE_VER=lts
+RUN curl -sL https://deb.nodesource.com/setup_${NODE_VER}.x | bash -
```

Comment:
we were using "lts" in the client build script, but "6.x" here, which became very outdated.